### DISCUSSION TITLE
Submitting Lab2 Companion Issues

### DISCUSSION THREAD

**Discussion by pkwia0001** [community/community#102880] [GitHub Education]
### Hi everyone,

Hello,

I am currently trying to submit my lab in GitHub and it just keeps loading and never displays a grade in MindTap-Cengage. I have looked around to submit some sort of ticket but if anyone can point me in the right direction I would greatly appreciate it. Thank you

**Comment by phuccoder2000** [comment]
Hi @pkwia0001 
I think u clear Browser Cache:
 - Cached data in your browser may sometimes cause issues. Try clearing your browser cache and cookies. This might resolve any caching-related problems.
 Good luck!

**Comment by mwhitted01** [comment]
The cengage companion on github just sucks.
